From 746d6210a3686b93490ce85b384f5f5906c9937a Mon Sep 17 00:00:00 2001
From: Dan Willemsen <dwillemsen@google.com>
Date: Fri, 26 Feb 2016 22:03:54 -0800
Subject: [PATCH 07/11] msm8974: Fix duplicate gps.conf for hammerhead
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since hammerhead includes gps.conf in device tree, don't
attempt to install it here as well.

Bug: 26294231
Change-Id: I837337170e8ee61876de01683a2f05b1f4350793
Grabbed via msm8960, to avoid similar warnings
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/etc/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/msm8974/etc/Android.mk b/msm8974/etc/Android.mk
index a6914a2..8511085 100644
--- a/msm8974/etc/Android.mk
+++ b/msm8974/etc/Android.mk
@@ -1,4 +1,6 @@
 LOCAL_PATH := $(call my-dir)
+
+ifeq ($(filter hammerhead,$(TARGET_DEVICE)),)
 include $(CLEAR_VARS)
 
 LOCAL_MODULE := gps.conf
@@ -8,3 +10,4 @@ LOCAL_MODULE_PATH  := $(TARGET_OUT_ETC)/
 LOCAL_SRC_FILES := gps.conf
 
 include $(BUILD_PREBUILT)
+endif
-- 
2.17.1

